Enterprise-CMCS/macpro-mdct-mfp
# [CMDCT-3145] Report period and year fix

### Description



https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/03f8681f-7f42-44d1-b7d9-65edf14ea622


### Related ticket(s)


CMDCT-3145

---
### How to test

Make sure that your isCopyOverTest is set to true on line 44 in AddEditReportModal `const isCopyOverTest = true`
Create a work plan and submit it as a state user
Approve that work plan as an admin user
Copy over the work plan as a state user
See that its been copied into the new year!

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
